content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+- fabriziopandini
+- luxas
+- timothysc 
+title: Customizing control plane flags with kubeadm
+---
+
+{{% capture overview %}}
+
+kubeadm’s configuration exposes the following fields that can be used to override the default flags passed to control plane components such as the APISever, ControllerManager and Scheduler:
+
+- `APIServerExtraArgs`
+- `ControllerManagerExtraArgs`
+- `SchedulerExtraArgs`
+
+The fields can contain a list of `key: value` pairs. To override flags for a specific control plane component just include one of the fields above and start adding flags for it.
```

Comment:
you can add a `(it's serialized as a map[string]string`, like Kubernetes labels or annotations)`

Reply:
s/just/

Reply:
What value for a user does the content that luxas suggests add? This seems like an implementation detail that does not belong here. We've already got a larger confusion in the kubeadm docs between what a user needs and what a dev needs; let's not perpetuate it here.

Reply:
Larger scale edit:

These fields consist of `key: value` pairs. To override a flag for a control plane component: 

1. Add the appropriate field to your configuration.
1. Add the flags to override to the field.